tsdb/mapper.go

```diff
@@ -171,20 +175,15 @@ func (lm *LocalMapper) Open() error {
 			return err
 		}
 		tagSets := tsf.tagSets
-		lm.selectFields = tsf.selectFields
```

Comment:
This was bug, whereby the set of fields and tags was being overwritten each loop of `Sources`.